packages/valory/skills/price_estimation_abci/tests/test_behaviours.py

```diff
@@ -528,6 +522,8 @@ def test_pack_for_server(
 def test_fuzz_pack_for_server() -> None:
     """Test fuzz pack_for_server."""
 
+    import atheris  # type: ignore
```

Comment:
I assuming the test is being skipped conditionally in environments where we cannot import atheris

Reply:
It is skipped everywhere:

https://github.com/valory-xyz/price-oracle/blob/bef125ea84ac59065de82af26b68a9f85ba64cf5/packages/valory/skills/price_estimation_abci/tests/test_behaviours.py#L521